gcp_genotype_storage/setup.py

```diff
@@ -0,0 +1,40 @@
+"""Setup for GPF genotype storage on the GCP."""
```

Comment:
As this is a new part of gpf I think you should drop a line in the README.md describing it. Something like:
```
If you also want support for GCP run
pip install -e gcp_genotype_storage
```

Reply:
fixed